packages/manager/modules/pci/src/projects/new/payment/components/add/controller.js

```diff
@@ -24,9 +25,16 @@ export default class PciProjectNewPaymentMethodAddCtrl {
     this.$location = $location;
     this.coreConfig = coreConfig;
     this.ovhPaymentMethodHelper = ovhPaymentMethodHelper;
-    this.registrationCharges = `${CHARGES}${
-      this.coreConfig.getUser().currency.code
-    }`;
+    const { currency, ovhSubsidiary } = this.coreConfig.getUser();
+    this.registrationCharges = new Intl.NumberFormat(
+      LANGUAGE_OVERRIDE[ovhSubsidiary]
+        ? LANGUAGE_OVERRIDE[ovhSubsidiary]
+        : ovhSubsidiary.toLowerCase(),
```

Comment:
I think we have to use coreConfig.getUserLocale().replace('_', '-') instead of the sub

Reply:
Nevermind, it seems to work fine with the sub

Reply:
We cannot use user locale. As fr user can also use `en_GB` locale. So it will not be appropriate.